### ISSUE TITLE
VSCode Sidekick Plugin LogPoint Encoding And Visualization Issues

### ISSUE THREAD

**Issue by probably-not** [runsidekick/sidekick-agent-nodejs#7]
I've added a LogPoint on a line that contains an exception (See #2 for the code example). The LogPoint logExpression is defined as `{{exception.stack}}`, which will output the exception stack for us to trace back to the original exception point.

The current output looks like this:
```
UnauthorizedException: Unauthorized at MixinAuthGuard.handleRequest (&#x2F;app&#x2F;node_modules&#x2F;@nestjs&#x2F;passport&#x2F;dist&#x2F;auth.guard.js:68:30) at &#x2F;app&#x2F;node_modules&#x2F;@nestjs&#x2F;passport&#x2F;dist&#x2F;auth.guard.js:49:128 at &#x2F;app&#x2F;node_modules&#x2F;@nestjs&#x2F;passport&#x2F;dist&#x2F;auth.guard.js:91:24 at allFailed (&#x2F;app&#x2F;node_modules&#x2F;passport&#x2F;lib&#x2F;middleware&#x2F;authenticate.js:110:18) at attempt (&#x2F;app&#x2F;node_modules&#x2F;passport&#x2F;lib&#x2F;middleware&#x2F;authenticate.js:183:28) at JwtStrategy.strategy.fail (&#x2F;app&#x2F;node_modules&#x2F;passport&#x2F;lib&#x2F;middleware&#x2F;authenticate.js:305:9) at JwtStrategy.authenticate (&#x2F;app&#x2F;node_modules&#x2F;passport-jwt&#x2F;lib&#x2F;strategy.js:96:21) at attempt (&#x2F;app&#x2F;node_modules&#x2F;passport&#x2F;lib&#x2F;middleware&#x2F;authenticate.js:369:16) at authenticate (&#x2F;app&#x2F;node_modules&#x2F;passport&#x2F;lib&#x2F;middleware&#x2F;authenticate.js:370:7) at &#x2F;app&#x2F;node_modules&#x2F;@nestjs&#x2F;passport&#x2F;dist&#x2F;auth.guard.js:96:3 at new Promise (&lt;anonymous&gt;) at &#x2F;app&#x2F;node_modules&#x2F;@nestjs&#x2F;passport&#x2F;dist&#x2F;auth.guard.js:88:83 at MixinAuthGuard.&lt;anonymous&gt; (&#x2F;app&#x2F;node_modules&#x2F;@nestjs&#x2F;passport&#x2F;dist&#x2F;auth.guard.js:49:36) at Generator.next (&lt;anonymous&gt;) at fulfilled (&#x2F;app&#x2F;node_modules&#x2F;@nestjs&#x2F;passport&#x2F;dist&#x2F;auth.guard.js:17:58) at runMicrotasks (&lt;anonymous&gt;) at processTicksAndRejections (node:internal&#x2F;process&#x2F;task_queues:96:5)
```

It looks like there's an encoding (HTML encoding I think?) that's not being decoded properly in VSCode's visualization.

This is what it should look like when I decode it manually:
```
UnauthorizedException: Unauthorized at MixinAuthGuard.handleRequest (/app/node_modules/@nestjs/passport/dist/auth.guard.js:68:30) at /app/node_modules/@nestjs/passport/dist/auth.guard.js:49:128 at /app/node_modules/@nestjs/passport/dist/auth.guard.js:91:24 at allFailed (/app/node_modules/passport/lib/middleware/authenticate.js:110:18) at attempt (/app/node_modules/passport/lib/middleware/authenticate.js:183:28) at JwtStrategy.strategy.fail (/app/node_modules/passport/lib/middleware/authenticate.js:305:9) at JwtStrategy.authenticate (/app/node_modules/passport-jwt/lib/strategy.js:96:21) at attempt (/app/node_modules/passport/lib/middleware/authenticate.js:369:16) at authenticate (/app/node_modules/passport/lib/middleware/authenticate.js:370:7) at /app/node_modules/@nestjs/passport/dist/auth.guard.js:96:3 at new Promise (<anonymous>) at /app/node_modules/@nestjs/passport/dist/auth.guard.js:88:83 at MixinAuthGuard.<anonymous> (/app/node_modules/@nestjs/passport/dist/auth.guard.js:49:36) at Generator.next (<anonymous>) at fulfilled (/app/node_modules/@nestjs/passport/dist/auth.guard.js:17:58) at runMicrotasks (<anonymous>) at processTicksAndRejections (node:internal/process/task_queues:96:5)

```

**Comment by boroskoyo** [comment]
Seems like we've missed a spot 👍 We'll fix this one asap

**Comment by hserefal** [comment]
Hi, we resolved the issue and published version `v0.0.18` of our vscode extension.  🙂
